src/resolvers/message.ts

```diff
@@ -1,25 +1,76 @@
-import { Message, Resolvers } from '../generated/graphql';
+import { Message, MessagesConnection, Resolvers } from '../generated/graphql';
+import { Op, WhereOptions } from 'sequelize';
 
 import { checkAuth } from '../utils/auth';
+import paginateResults from '../utils/pagination';
 
 const resolver: Resolvers = {
   Query: {
     messages: async (
       _,
-      args, {
-        verifyUser,
-        models,
-      },
-    ): Promise<Message[]> => {
+      args,
+      { verifyUser, models },
+    ): Promise<MessagesConnection> => {
       const { Message: messageModel } = models;
       const auth = verifyUser();
       checkAuth(auth);
 
-      return messageModel.findAll({
-        where: {
-          messageId: 1,
-        },
+      const { message, channelId, pageSize = 20, after } = args;
+      let qryMsg: object = {};
+      let qryAfter: object = {};
+      let qryChannelId: object = {};
+
+      if (message) {
+        const msgOpConds: object = {};
+        const findColName = 'text';
+        msgOpConds[findColName] = {
```

Comment:
Why are you adding `text` here and make a `msgOpConds`?

Reply:
I think `messages` resolver doesn't have any practical code so I randomly have put the code similar to `users` resolver for testing pagination.
and I wanted to test out to find message by `text`.